dash-renderer/webpack.base.config.js

```diff
@@ -0,0 +1,85 @@
+const webpack = require('webpack');
+const R = require('ramda');
+const path = require('path');
+const packagejson = require('./package.json');
+const dashLibraryName = packagejson.name.replace(/-/g, '_');
+
+const defaults = {
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /node_modules/,
+                use: {
+                    loader: 'babel-loader',
+                },
+            },
+            {
+                test: /\.ts(x?)$/,
+                exclude: /node_modules/,
+                use: ['babel-loader', 'ts-loader'],
+            },
+            {
+                test: /\.css$/,
+                use: ['style-loader', 'css-loader'],
+            },
+            {
+                test: /\.svg$/,
+                use: ['@svgr/webpack'],
+            }
+        ]
+    },
+    resolve: {
+        extensions: ['.js', '.ts', '.tsx']
+    }
+};
+
+const rendererOptions = {
+    mode: 'development',
+    entry: {
+        main: ['whatwg-fetch', './src/index.js'],
+    },
+    output: {
+        path: path.resolve(__dirname, dashLibraryName),
+        filename: `${dashLibraryName}.dev.js`,
+        library: dashLibraryName,
+        libraryTarget: 'window',
+    },
+    externals: {
+        react: 'React',
+        'react-dom': 'ReactDOM',
+        'prop-types': 'PropTypes'
+    },
+    ...defaults
+};
+
+module.exports = options => [
```

Comment:
Small change here allowing further customization of the Webpack configuration for (a) the build, (b) testing